README.md

```diff
@@ -1 +1,13 @@
-# simon-says
\ No newline at end of file
+# simon-says
+
+## Order
+
+1. Build the html
+2. Make the buttons light up when you click them
+3. Play a hardcoded pattern like ['red', 'green', 'blue']
+4. Handle win and lose for hardcoded pattern and user input
+5. Generate a random pattern
+6. Block user input during pattern play
```

Comment:
Worth doing in my opinion. Have a Boolean toggle in the state object is a pattern that pops up so often that they may as well see a few different examples.